react/features/virtual-background/components/VirtualBackgroundDialog.js

```diff
@@ -59,6 +60,11 @@ type Props = {
      */
     _virtualBackground: Object,
 
+    /**
+    * Whether or not multi-stream send support is enabled.
+    */
+    _multiStreamModeEnabled: boolean,
```

Comment:
style:alpha sort

Reply:
Well, the rest were not alpha sorted so I added it to the end of the list :)